frontend/src/component/releases/ReleasePlanTemplate/ReleasePlanTemplateAddStrategyForm.tsx

```diff
@@ -148,6 +150,10 @@ export const ReleasePlanTemplateAddStrategyForm = ({
             ? String(addStrategy.parameters.stickiness)
             : 'default';
 
+    useEffect(() => {
+        setSegments([]);
+    }, []);
+
```

Comment:
Separate task - Segments doesn't work yet. Need to map existing segments here, but the hooks in use don't work with release plans/templates